crates/type-system/src/ontology/entity_type/mod.rs

```diff
@@ -43,16 +43,7 @@ impl EntityType {
         default: HashMap<BaseUri, serde_json::Value>,
         examples: Vec<HashMap<BaseUri, serde_json::Value>>,
     ) -> Self {
-        Self {
-            id,
-            title,
-            plural_title,
-            description,
-            default,
-            examples,
-            property_object,
-            links,
-        }
+        Self { id, title, plural_title, description, property_object, links, default, examples }
```

Comment:
Are you sure this formatting is correct? I've got the change locally to re-order and fix the lint but:
```rust
    ) -> Self {
        Self {
            id,
            title,
            plural_title,
            description,
            property_object,
            links,
            default,
            examples,
        }
    }
```

Reply:
This is the clippy error:
<img width="950" alt="image" src="https://user-images.githubusercontent.com/21277928/188119009-3cc70d15-aaa5-48fb-94e9-dbe43c20f231.png">

Reply:
Yeah I know as I said I've fixed it locally
> I've got the change locally to re-order and fix the lint but:

My point is your commit puts them all on the same line. That's why I'm asking if you're sure the **formatting** is correct